WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -10182,6 +10162,14 @@ from anywhere.";
 /* Name of the "Save as Draft" action as it should appear in the iOS Share Sheet when sharing content from other apps to WordPress. Must be less than 16 characters long. Typically the same text as CFBundleDisplayName, but could be shorter if needed. */
 "ios-sharesheet.CFBundleName" = "Save as Draft";
 
+/* MARK: - InfoPlist.strings */
+
+/* Name of the "Save to Jetpack" action as it should appear in the iOS Share Sheet when sharing content from other apps to WordPress */
+"ios-jetpack-sharesheet.CFBundleDisplayName" = "Save to Jetpack";
+
+/* Name of the "Save to Jetpack" action as it should appear in the iOS Share Sheet when sharing content from other apps to WordPress. Must be less than 16 characters long. Typically the same text as CFBundleDisplayName, but could be shorter if needed. */
+"ios-jetpack-sharesheet.CFBundleName" = "Save to Jetpack";
```

Comment:
These are new entries from 26dd24c878ba63aaa161a0fcc3c35d295d7cdb5e.

The `/* MARK: – InfoPlist.string */` header is helpful in separating these strings from those coming from other files, but also a bit too generic–There is another `MARK` header with the same file name at line 10157 before this one.

I wonder if we should look into ways to better name the headers, or if relying on the file name plus the different prefixes is enough to help us navigate this file.

Reply:
Good point. The question though would be, what could be a better header that would still be automatically generated? (i.e. where would the automation deduce the name of the header from if not just the file name?) Or maybe having the header say `/* MARK: - InfoPlist.strings (ios-jetpack-sharesheet.) */` would already be enough to improve things?

PS: We could also imagine improving our action that merges iOS strings files to provide custom headers chosen ourselves (in addition to the custom prefixes that we already specify explicitly), but the API might not be super pretty (because we already have a `Hash` of `File => prefix` for that, so where would we provide the header name too in that Hash without making the API too ugly? Might not be worth it.